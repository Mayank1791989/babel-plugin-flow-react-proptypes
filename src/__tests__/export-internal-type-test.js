const babel = require('babel-core');
const content = `
/* @flow */
type Test = { id: string };
export type { Test };
`;

it('basic', () => {
  const res = babel.transform(content, {
    babelrc: false,
    presets: ['es2015', 'stage-1', 'react'],
    plugins: ['syntax-flow', require('../')]
  }).code;
  expect(res).toMatchSnapshot();
});
