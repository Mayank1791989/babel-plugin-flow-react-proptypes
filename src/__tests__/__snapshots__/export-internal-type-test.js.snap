// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic 1`] = `
"\\"use strict\\";

Object.defineProperty(exports, \\"__esModule\\", {
  value: true
});
exports.bpfrpt_proptype_Test = undefined;

var _propTypes = require(\\"prop-types\\");

var _propTypes2 = _interopRequireDefault(_propTypes);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var bpfrpt_proptype_Test = {
  id: _propTypes2.default.string.isRequired
};
exports.bpfrpt_proptype_Test = bpfrpt_proptype_Test;"
`;
